docs/docs/tutorial/android.md

```diff
@@ -0,0 +1,19 @@
+---
+sidebar_position: 9
+sidebar_label: Targetting Android
+title: Targetting Android
+---
+
+This tutorial explains how to set up the plugin to target Android devices.
+
+## Prerequisites
+- JDK 17 or later
+
+## Setup
+1. Follow the [installation](installation) instructions for your development platform.
+2. Make sure that the Unreal Editor is closed.
+3. Download a plugin [release](https://github.com/DolbyIO/comms-sdk-unreal/releases) for Android and unpack the plugin to the `Plugins` folder. The release version should match your development platform release. Say yes to any overwrite prompts.
+4. Edit `{UnrealEngineRoot}/Build/Android/Java/gradle/gradle/wrapper/gradle-wrapper.properties` and modify `distributionUrl` to `https\://services.gradle.org/distributions/gradle-7.4.1-all.zip`
```

Comment:
Is it possible to get rid of this step? Can changing the engine's Gradle distribution URL make other plugins not work?

Reply:
We can't get rid of it. The distribution url version is required due. I don't see plugins where this would be a problem. I tested few like the store etc... but again, remember that unreal is actually an unstable half baked solution and that even today, I'm quite impressed it didn't fell apart already regarding Android (haven't dug much into the actual reasons behind that)